contracts/bonding/BondingManager.sol

```diff
@@ -783,26 +816,28 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     }
 
     /**
-     * @dev Return whether a transcoder is registered
+     * @notice Return whether a transcoder is registered
      * @param _transcoder Transcoder address
+     * @return true if transcoder is self-bonded
      */
     function isRegisteredTranscoder(address _transcoder) public view returns (bool) {
         Delegator storage d = delegators[_transcoder];
         return d.delegateAddress == _transcoder && d.bondedAmount > 0;
     }
 
     /**
-     * @dev Return whether an unbonding lock for a delegator is valid
+     * @notice Return whether an unbonding lock for a delegator is valid
      * @param _delegator Address of delegator
      * @param _unbondingLockId ID of unbonding lock
+     * @return true if unbondingLock for ID has a non-zero withdraw round
      */
     function isValidUnbondingLock(address _delegator, uint256 _unbondingLockId) public view returns (bool) {
         // A unbonding lock is only valid if it has a non-zero withdraw round (the default value is zero)
         return delegators[_delegator].unbondingLocks[_unbondingLockId].withdrawRound > 0;
     }
 
     /**
-     * @dev Increase the total stake for a delegate and updates its 'lastActiveStakeUpdateRound'
+     * @notice Increase the total stake for a delegate and updates its 'lastActiveStakeUpdateRound'
```

Comment:
Should these helper use `@dev` tags since they are internal functions and not user facing? Same comment for other internal functions in this file.

Reply:
the dangers of find&replace I guess , fixed in 8d0bb3e